app/views/install/compose.phtml

```diff
@@ -119,6 +119,10 @@ services:
       - _APP_STORAGE_DO_SPACES_SECRET
       - _APP_STORAGE_DO_SPACES_REGION
       - _APP_STORAGE_DO_SPACES_BUCKET
+      - _APP_STORAGE_BACKBLAZE_ACCESS_KEY
```

Comment:
This list is becoming very long, lets try to use the same syntax as use for `x-logging` in our compose files.

Reply:
addressed